src/conf.c

```diff
@@ -278,6 +279,18 @@ account_t *find_account(list_t *acc_list, const char *id)
     return a;
 }
 
+static bool from_matches_account_from(const char *from, const char *acc_from)
+{
+#ifdef HAVE_FNMATCH_H
+    if (strchr(acc_from, '?') || strchr(acc_from, '*') || strchr(acc_from, '['))
```

Comment:
Hmm, I could have used `strpbrk(acc_from, "?*+")` here, to search for any of the wildcard characters in a single pass through the string. Typical addresses are probably not long enough that the micro-optimization matters.